Kudu.Core/Tracing/KuduEventSource.cs

```diff
@@ -42,5 +42,41 @@ public void DeprecatedApiUsed(string siteName, string route, string userAgent, s
                 WriteEvent(65510, siteName, route, userAgent, method, path);
             }
         }
+
+        [Event(65511, Level = EventLevel.Informational, Message = "SiteExtension {1} installation started for site {0}", Channel = EventChannel.Operational)]
+        public void SiteExtensionInstallStarted(string siteName, string siteExtensionName, string siteExtensionVersion, string siteExtensionFeedUrl, string type)
+        {
+            if (IsEnabled())
+            {
+                WriteEvent(65511, siteName, siteExtensionName, siteExtensionVersion, siteExtensionFeedUrl, type);
+            }
+        }
+
+        [Event(65512, Level = EventLevel.Informational, Message = "SiteExtension {1} installation finished for site {0}", Channel = EventChannel.Operational)]
+        public void SiteExtensionInstallEnded(string siteName, string siteExtensionName, string operation, string httpStatus, string provisionState, string comment)
+        {
+            if (IsEnabled())
+            {
+                WriteEvent(65512, siteName, siteExtensionName, operation, httpStatus, provisionState, comment);
+            }
+        }
+
+        [Event(65513, Level = EventLevel.Informational, Message = "SiteExtension {1} uninstallation started for site {0}", Channel = EventChannel.Operational)]
+        public void SiteExtensionUninstallStarted(string siteName, string siteExtensionName)
+        {
+            if (IsEnabled())
+            {
+                WriteEvent(65513, siteName, siteExtensionName);
+            }
+        }
+
+        [Event(65514, Level = EventLevel.Informational, Message = "SiteExtension {1} uninstallation finished for site {0}", Channel = EventChannel.Operational)]
+        public void SiteExtensionUninstallEnded(string siteName, string siteExtensionName)
+        {
+            if (IsEnabled())
+            {
+                WriteEvent(65514, siteName, siteExtensionName);
+            }
+        }
```

Comment:
if this new event in this PR are good, i will update website event config file to add the four new event.(will be a separate code review.)